src/component/handlers/drag/DraftEditorDragHandler.js

```diff
@@ -23,6 +23,7 @@ const getUpdatedSelectionState = require('getUpdatedSelectionState');
 const nullthrows = require('nullthrows');
 
 import type SelectionState from 'SelectionState';
+const isEventHandled = require('isEventHandled');
```

Comment:
@guitardave24 see my comment below. Never had issues with es6 modules before but here the build process just didn't find the module when using `import`.